lib/scrivener/html.ex

```diff
@@ -240,11 +254,13 @@ defmodule Scrivener.HTML do
     add_first(paginator.page_number, options[:distance], options[:first])
     |> add_previous(paginator.page_number)
     |> page_number_list(paginator.page_number, paginator.total_pages, options[:distance])
+    |> add_ellipsis(paginator.page_number, paginator.total_pages, options[:distance], options[:ellipsis])
```

Comment:
We may need to make an `add_first_ellipsis/5` and `add_last_ellipsis/5` to provide ellipsis on both ends as the math will be different depending upon which end.